src/App.jsx

```diff
@@ -1,7 +1,10 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import cn from 'classnames';
+import { useState } from 'react';
+import { GoodsList } from './components/GoodsList/GoodsList';
```

Comment:
Need to add this file to the Pull Request
<img width="1266" alt="image" src="https://github.com/mate-academy/react_list-of-goods-js/assets/64750958/feb104d9-7391-4c4c-96bc-4e622a279a68">